packages/wrangler/src/__tests__/r2.test.ts

```diff
@@ -290,6 +290,29 @@ describe("wrangler", () => {
 			Upload complete."
 		`);
 			});
+
+			it("should delete R2 object from bucket", async () => {
+				setMockFetchR2Objects({
+					accountId: "some-account-id",
+					bucketName: "bucketName-object-test",
+					objectName: "wormhole-img.png",
+				});
+
+				await runWrangler(
+					`r2 object delete bucketName-object-test/wormhole-img.png`
+				);
+
+				expect(std).toMatchInlineSnapshot(`
+			Object {
+			  "debug": "",
+			  "err": "",
+			  "out": "Deleting object \\"wormhole-img.png\\" from bucket \\"bucketName-object-test\\".
+			Delete complete.",
+			  "warn": "",
+			}
+		`);
```

Comment:
For consistency, can we make this `expect(std.out)` instead of `expect(std)`?

Reply:
Sure.